images/conduit/start.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Start conduit daemon. There are various configurations controlled by
+# environment variables.
+#
+# Configuration:
+#   DISABLED          - If set start a server that returns an error instead of conduit.
+#   CONNECTION_STRING - the postgres connection string to use.
+#   ALGOD_ADDR        - host:port to connect to for algod follower.
+#   ALGOD_ADMIN_TOKEN - the algod admin token.
+#   PORT              - port to start conduit on.
+set -e
+
+THIS=$(basename "$0")
+
+disabled() {
```

Comment:
I see that this pattern does make our assumptions in the `sandbox` script more uniform. I still consider it non-ideal to stand up docker images that are just zombies. For example doing a `docker ps` you might believe that indexer/conduit are actually running. 

If others agree that this is tech debt and should eventually be addressed, I can create an issue.

Reply:
Unless we can remove them while simplifying sandbox, I'm not convinced it's worth the effort. I'd like to address this as part of a redesign which is built around the public docker images, I think there are a few steps to get there which start with some improvements to the existing docker images.